src/tui.rs

```diff
@@ -403,6 +409,18 @@ impl ToipeTui {
         //   - won't hang there waiting for user to type space
         lines.push(Text::from(line.join(" ")).with_faint());
 
+        if lines.len() + self.bottom_lines_len + 1 >= terminal_height.into() {
```

Comment:
The `into()` here is to convert `u16` into `usize`, right? If so, it would be better to do `as usize` since it's used in the rest of the code.

Reply:
yeah, it is. i changed it to `as usize`.